### PR TITLE
[Symfony] Handle property exists on ContainerGetToConstructorInjectionRector

### PR DESCRIPTION
**Pull request by samsonasik**
Fixes https://github.com/rectorphp/rector-symfony/issues/111

### CODE REVIEW

File: src/NodeAnalyzer/DependencyInjectionMethodCallAnalyzer.php [hunk 1]
```diff
@@ -38,10 +43,32 @@ public function replaceMethodCallWithPropertyFetchAndDependency(MethodCall $meth
...
     }
+
+    private function resolveNewPropertyNameWhenExists(Class_ $class, string $propertyName, int $count = 1): string
+    {
+        $promotedPropertyParams = $this->promotedPropertyResolver->resolveFromClass($class);
+        foreach ($promotedPropertyParams as $promotedPropertyParam) {
+            if ($this->nodeNameResolver->isName($promotedPropertyParam->var, $propertyName)) {
+                ++$count;
+                $propertyName .= $count;
+                return $this->resolveNewPropertyNameWhenExists($class, $propertyName, $count);
```
Comment (samsonasik) [inline src/NodeAnalyzer/DependencyInjectionMethodCallAnalyzer.php:61]: current `VariableNaming` count++ naming only handle variable, so use recursive check here to check property exists.

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

**Review by TomasVotruba — APPROVED** [review]

**Comment by TomasVotruba** [comment]
Thank you

**Pull request merged**